### DISCUSSION TITLE
Which features are available when running over SSH?

### DISCUSSION THREAD

**Discussion by kevinchannon** [ArthurSonzogni/FTXUI#775] [Q&A]
I can imagine a lot of things would work fine, but is there a list of things to avoid if you're running an app based on this framework over an SSH connection to some server, or something?

The mouse interactions seem like they'd probably be a stretch, for example?

**Comment by ArthurSonzogni** [comment]
Hi @kevinchannon,
I am expecting everything to behave correctly through ssh, including mouse support, on unix terminals.

After all, everything is only about reading from stdin and writing into stout. As long as ssh is forwarding thing correctly, it should work well.

Maybe some discrepancy in between the local terminal emulator and the remote one? Maybe  the colors supported aren't matching? So far, everything was working as expected on my side.

**Reply by kevinchannon** [reply]
Thanks!
